pkg/logger/logger.go

```diff
@@ -0,0 +1,43 @@
+package logger
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/adrg/xdg"
+	"go.uber.org/zap"
+)
+
+var logFile = fmt.Sprintf("october/logs/%s.log", time.Now().Format("2006-01-02"))
+var Log *zap.SugaredLogger
```

Comment:
This way of having a global logger seems dumb but it would be more annoying passing around the logger everywhere. How does logrus do it with the Global() call? How does that work with testing?